lib/api.es6.js

```diff
@@ -8,7 +8,7 @@ const SCOPES = 'history,identity,mysubreddits,read,subscribe,vote,submit,' +
                'modwiki,privatemessages,report,wikiedit,wikiread';
 
 
-const login = (apiOptions, username, pass) => {
+const login = (apiOptions, username, pass, orderedHeaders, clientUserAgent) => {
```

Comment:
modmail also uses @r/private. I know there isn't anything we have to change immediately cause of semvar, but if modmail _did_ want to upgrade r/private, are passing along the ordered headers ok? or, should we make this optional somehow?